nasa/earthdata-search
# EDSC-2927: Panel Redesign

# Overview

### What is the feature?

Redesigns the panel layout

### What is the Solution?

The site header was moved out of the sidebar and into a global header, adding the functionality to pull in the users first name. The PanelGroup and PanelGroupHeader components were updated to standardize the repeated functionality between the headers of the different routes (title, loading states, breadcrumbs, view, sort, etc). Granule Search has been moved into the sidebar along with the rest of the granule filters.

### What areas of the application does this impact?

 - Global layout (header/sidebar)
 - Search route
 - Project route

# Testing

### Reproduction steps

- **Environment for testing:** Any
- **Collection to test with:** Any

1. Visit the site
2. Navigate to /search, /search/granules, /search/granules/collection-details, /search/granules/granule-details, /projects routes and confirm intended behavior
3. Navigate to all other site pages and confirm unchanged behavior

# Checklist
